### PR TITLE
navbar now takes props for links

### PR DESCRIPTION
**Pull request by guy-ro**
Navbar takes props of the following form:
image

Donor page has the following prop for the navbar:
image

### CODE REVIEW

File: client/src/components/layouts/DonorPage.tsx [hunk 1]
```diff
@@ -5,9 +5,17 @@ import Navbar from '../organisms/Navbar'
 
 const DonorPage: FunctionComponent<Record<string, unknown>> = (props: React.PropsWithChildren<Record<string, unknown>>) => {
   return <div className="donor-page">
-    <div className="donor-page-header"><Navbar /></div>
-    <div className="donor-page-content" >{props.children}</div>
-    <div className="donor-page-footer"><Footer /></div>
+    <div className="donor-page-header">
+      <Navbar links={ [ { name: "TPC Main Website", link: "/donation-guidelines"  },
```
Comment (meganniu) [inline client/src/components/layouts/DonorPage.tsx]: Can we link "TPC Main Website" to `https://pregnancycentre.ca/`?

File: client/src/components/organisms/Navbar.tsx [hunk 2]
```diff
@@ -13,6 +22,9 @@ const Navbar: FunctionComponent = () => {
       />
     </BsNavbar.Brand>
     <div className="navbar-links">
+      {props.links.map((link : Link) => {
+        <Nav.Link href={link.link}>{link.name}</Nav.Link>
```
Comment (meganniu) [inline client/src/components/organisms/Navbar.tsx]: I think each `Nav.Link` needs a key prop (otherwise React complains)

**Review by meganniu — APPROVED** [review]
LGTM!!! thanks :)))

To confirm, the homepage navbar and old `Nav.Link`s will be removed in another PR?

**Review by jlight99 — APPROVED** [review]
looks good!

File: client/src/components/layouts/DonorPage.tsx (hunk 1 above)
Comment (guy-ro) [reply client/src/components/layouts/DonorPage.tsx]: done

**Pull request merged**